module-1/lab-data_cleaning/your-code/main_lab_data_cleaning_miriam.ipynb

```diff
@@ -0,0 +1,3583 @@
+{
```

Comment:
1. When you need to check the output of the execution of some stuff done on a dataframe with a print instruction, you only need to print the head or the tail of the dataframe (depending what you need to check). When you see that your dataframe is printed in a non fashion way, try to use the display function. 
2. :
###posts['userId'] = posts['userId'].astype('int64')
###posts['userId'].dtype
### Erro, no puede convertir NaN en int64 
You have decided to fillna with zeros before changing the data type. Good approach!! If you were working in the real world, you can invert the order of the steps depending on what you find out in your exploratory phase. 
3. Step 9: You have decided to drop the missing values because there are more than 50% of the dataset. The rational is the other way around, data cleaning is one of the first steps before modeling stage. This time the number of observations could be considered big enough for further data science process, but it's just a coincidence. 
4. Step 11: I would should the explanation about the reasons to change this field data type and why you have inverted some of the steps on your notebook.